src/main/kotlin/study/racing/car/Car.kt

```diff
@@ -0,0 +1,15 @@
+package study.racing.car
+
+class Car(
+    val name: String,
+    totalMoved: Int? = 0
+) {
+    var totalMoved = totalMoved ?: 0
+        private set
+
+    fun move(condition: Int) {
+        if (condition >= 4) {
+            totalMoved += 1
+        }
+    }
```

Comment:
[이전 단계 리뷰](https://github.com/next-step/kotlin-racingcar/pull/1139/files#r1225810429)에서도 말씀드렸지만, 지금은 이동 조건이 변경되면 Car 객체도 함께 변경될 것 같아요.

`정지, 전진`를 결정하는 책임을`이동 조건`이라는 객체가 담당한다면, `Car`는 0~ 3은 정지, 4 ~9는 이동이라는 세부적인 이동 조건에 대해 모르더라도 `이동 조건`이라는 객체와 협력해 이동할 수 있지 않을까요??
예를 들면 아래와 같은 형태로 표현할 수 있을 것 같아요.

```suggestion
    fun move(movingCondition: MovingCondition) {
         totalMoved += movingCondition.drive()
     }
```